GVRf/Extensions/gvrf-physics/src/main/java/org/gearvrf/physics/GVRPhysicsLoader.java

```diff
@@ -46,7 +46,8 @@
      * @param fileName Physics settings file name.
      * @param scene The scene containing the objects to attach physics components.
      */
-    public static void loadPhysicsFile(GVRContext gvrContext, String fileName, GVRScene scene) {
+    public static void loadPhysicsFile(GVRContext gvrContext, String fileName, GVRScene scene) throws IOException
```

Comment:
gvr-physicsload no longer compiles because this method now "throws IOException" where it didn't before.  Either this methods needs to not throw IOException, or the demo needs to be updated to catch it.

Reply:
it looks like the "throws IOException" is added back in.  does gvr-physicsload still compile?

Reply:
Yes it is working and the fix was made in the demo not in the framework.